test/unit/spec/common/jobCommMessages-spec.js

```diff
@@ -0,0 +1,25 @@
+define(['common/jobCommMessages'], (JobCommMessages) => {
+    'use strict';
+
+    describe('the JobCommMessages module', () => {
+        const expectedKeys = ['CHANNEL', 'MESSAGE_TYPE', 'PARAM', 'REQUESTS', 'RESPONSES'];
```

Comment:
Some basic tests for the JobCommMessages module that will also check the integrity of the job_config.json file.